GenderPayGap.WebUI/Models/Report/ReportFiguresViewModel.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using GenderPayGap.WebUI.Views.Models.ValidationAttributes;
+using GovUkDesignSystem;
+using GovUkDesignSystem.Attributes;
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace GenderPayGap.WebUI.Models.Report
+{
+    public class ReportFiguresViewModel : GovUkViewModel
+    {
+
+        [BindNever /* Output Only - only used for sending data from the Controller to the View */]
+        public Database.Organisation Organisation { get; set; }
+        [BindNever /* Output Only - only used for sending data from the Controller to the View */]
+        public int ReportingYear { get; set; }
+        [BindNever /* Output Only - only used for sending data from the Controller to the View */]
+        public bool IsEditingSubmittedReturn { get; set; }
+        
+        public DateTime SnapshotDate { get; set; }
+        
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Employer opted out of reporting pay quarter",
+            NameWithinSentence = "employer opted out of reporting pay quarter")]
+        public bool OptedOutOfReportingPayQuarters { get; set; }
+
+        #region Bonus Pay
+
+        [GovUkDisplayNameForErrors(
+            NameAtStartOfSentence = "Females who received bonus pay",
+            NameWithinSentence = "females who received bonus pay")]
+        [GovUkValidateDecimalRange(Minimum = 0, Maximum = 100)]
+        [DisplayFormat(DataFormatString = "{0:0.#}")]
```

Comment:
this doesn't work as expected - the figures are displayed with 2 decimal places

Reply:
@xonde this happens when I edit an already submitted return (submitted before you made this changes)